### PR TITLE
[Workers] Gradual Deployments edits

### PR DESCRIPTION
**Pull request by tanushree-sharma**

### CODE REVIEW

File: content/workers/configuration/versions-and-deployments/_index.md [hunk 1]
```diff
@@ -89,8 +89,27 @@ To view your deployments in the Cloudflare dashboard:
...
 
 Service worker syntax is not supported for versions that are uploaded through [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload). You must use ES modules format. 
 
-Refer to [Migrate from Service Workers to ES modules](/workers/reference/migrate-to-module-workers/#advantages-of-migrating) to learn how to migrate your Workers from the service worker format to the ES modules format.
\ No newline at end of file
+Refer to [Migrate from Service Workers to ES modules](/workers/reference/migrate-to-module-workers/#advantages-of-migrating) to learn how to migrate your Workers from the service worker format to the ES modules format.
+
+### Smart Placement 
+
+Smart Placement is not support for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you have Smart Placement enabled on a script.
```
Comment (deadlypants1973) [inline content/workers/configuration/versions-and-deployments/_index.md]:
```suggestion
Smart Placement is not supported for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you have Smart Placement enabled on a Worker. 
```

File: content/workers/configuration/versions-and-deployments/_index.md [hunk 2]
```diff
@@ -89,8 +89,27 @@ To view your deployments in the Cloudflare dashboard:
...
+
+### Smart Placement 
+
+Smart Placement is not support for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you have Smart Placement enabled on a script. 
+
+Smart Placement support for versions will be coming in the near future.
+
+### mTLS bindings 
+
+[mTLS bindings](/workers/runtime-apis/bindings/mtls/) are not support for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you mTLS bindings on a script.
```
Comment (deadlypants1973) [inline content/workers/configuration/versions-and-deployments/_index.md]:
```suggestion
[mTLS bindings](/workers/runtime-apis/bindings/mtls/) are not supported for versions that are uploaded through [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload). You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you configured mTLS bindings in your Worker. 
```

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -2133,15 +2133,14 @@ Retrieve your user information and test your authentication configuration.
...
-
 ---
 ## `versions`
 
 [Versions](/workers/configuration/versions-and-deployments/#versions) are currently in closed beta. Report bugs in [GitHub](https://github.com/cloudflare/workers-sdk/issues/new/choose).
 
 {{<Aside type="warning">}}
 
-The `--experimental-versions` flag is required to use the `wrangler versions` commands. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned.
+The `--experimental-versions` flag is required to use the `wrangler versions` commands. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned. For consistency in wrangler's output, it's recommended that you use the flag for all commands where it is an option.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
The `--experimental-versions` flag is required to use the `wrangler versions` commands. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned. For consistency in Wrangler's output, it is recommended that you use the `--experimental-versions` flag for all commands where it is an option. 
```

File: content/workers/wrangler/commands.md [hunk 4]
```diff
@@ -2231,7 +2230,7 @@ This command is currently in closed beta. Report bugs in [GitHub](https://github
 
 {{<Aside type="warning">}}
 
-The `--experimental-versions` flag is required to use the `wrangler triggers` commands. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned.
+The `--experimental-versions` flag is required to use the `wrangler triggers` commands. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned.  For consistency in wrangler's output, it's recommended that you use the flag for all commands where it is an option.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
The `--experimental-versions` flag is required to use the `wrangler triggers` commands. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned. For consistency in Wrangler's output, it is recommended that you use the `--experimental-versions` flag for all commands where it is an option. 
```

File: content/workers/wrangler/commands.md [hunk 5]
```diff
@@ -2308,7 +2307,7 @@ wrangler deployments list [OPTIONS]
 
 {{<Aside type="note">}}
 
-The `--experimental-versions` flag is required to use the new commands below. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned.
+The `--experimental-versions` flag is required to use the new commands below. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned.  For consistency in wrangler's output, it's recommended that you use the flag for all commands where it is an option.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
The `--experimental-versions` flag is required to use the new commands below. You may use the shorthand `--x-versions` flag in place of `--experimental-versions` anywhere it is mentioned. For consistency in Wrangler's output, it is recommended that you use the `--experimental-versions` flag for all commands where it is an option. 
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/workers/configuration/versions-and-deployments/_index.md [hunk 6]
```diff
@@ -89,8 +89,27 @@ To view your deployments in the Cloudflare dashboard:
 
 ## Limits
 
+### First upload
+
+You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions deploy --experimental-versions`](/workers/wrangler/commands/#upload) the first time you deploy a Worker will fail.
```
Comment (deadlypants1973) [inline content/workers/configuration/versions-and-deployments/_index.md]:
```suggestion
You must use [C3's deployment option](/workers/get-started/guide/#1-create-a-new-worker-project) to deploy your Worker or deploy with [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload) the first time you deploy a Worker will fail.
```
This should not be `wrangler versions deploy`, right? Assuming you mean `upload`?

Comment (tanushree-sharma) [inline content/workers/configuration/versions-and-deployments/_index.md]:
```suggestion
You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload) the first time you upload a Worker will fail.
```

**Pull request merged**